### DISCUSSION TITLE
Multi device configuration

### DISCUSSION THREAD

**Discussion by ramarivera** [Vinzent03/obsidian-git#695] [Q&A] [answered]
I currently am using Obsidian in two computers and two mobile (android) devices, plus using Obsidian Sync to keep them all synchronized.
I would like to set up Obsidian Git, so that only one of my computers is in charge of committing and pushing to the remote repo. Is this possible, by for example not completing the setup on all other devices?
I also saw the option "Disable on this device" but I am unsure how that works together with Obsidian Sync.

Thanks in advance for your time!

**Comment by Vinzent03** [comment, marked as answer]
I've added a Obsidian Sync section to the [docs](https://publish.obsidian.md/git-doc/Tips-and-Tricks#Usage+with+Obsidian+Sync). Does it clarify your setup?

**Reply by ramarivera** [reply]
Yeah, it clarifies it perfectly. Thanks!
